data/data/bootstrap/systemd/units/release-image.service.template

```diff
@@ -6,4 +6,7 @@ After=network-online.target
 [Service]
 Type=oneshot
 ExecStart=/usr/local/bin/release-image-download.sh
+{{if .IsOKD -}}
+ExecStart=/usr/local/bin/bootstrap-pivot.sh
```

Comment:
What is the reasoning behind adding this to the release-image.service rather than having it as a separate service?

Reply:
Most services like crio/kubelet are waiting for `release-image.service` to complete before starting.
Also log bundle collects logs from this service, so OKD log bundles would have pivot logs there too.

Perhaps its worth introducing `release-image.target` instead?

Reply:
Makes sense. Let's stick with what you have here for now, and we can consider other approaches later.